src/T2DMap.cpp

```diff
@@ -4611,13 +4611,42 @@ void T2DMap::wheelEvent(QWheelEvent* e)
     if (e->modifiers() & Qt::ControlModifier) { // Increase rate 10-fold if control key down - it makes scrolling through
                                                 // a large number of items in a listwidget's contents easier AND this make it
                                                 // easier to zoom in and out on LARGE area maps
-        delta *= 10;
+        delta *= 5;
     }
     if (delta != 0) {
         mPick = false;
-        int oldZoom = xyzoom;
-        xyzoom = qMax(3, xyzoom + delta);
+        qreal oldZoom = xyzoom;
+        xyzoom = qMax(3.0, xyzoom * pow(1.07, delta));
+
         if (oldZoom != xyzoom) {
```

Comment:
It is bad/wrong to test for equality/inequality on *floating-point* numbers - consider `qFuzzyCompare(...)` I think it is - instead!

Reply:
See other comment; while I can certainly change that, in this case it's at worst slightly suboptimal when you touch the 3.0 barrier.